client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -1,5 +1,6 @@
 import config from '@automattic/calypso-config';
 import { translate } from 'i18n-calypso';
+import { urlToSlug } from 'calypso/lib/url';
```

Comment:
`urlToSlug` will convert URLs like `my-main-site/second-site` to `my-main-site::second-site`.